deploy/helm/sumologic/templates/_helpers/_metrics.tpl

```diff
@@ -309,3 +309,22 @@ Example Usage:
 {{- define "metrics.collector.autoscaling.enabled" -}}
 {{- template "is.autoscaling.enabled" (dict "autoscalingEnabled" .Values.sumologic.metrics.collector.otelcol.autoscaling.enabled "Values" .Values) -}}
 {{- end -}}
+
+{{/*
+Returns list of namespaces to exclude
+
+Example:
+
+{{ include "metrics.excludeNamespaces" . }}
+*/}}
+{{- define "metrics.excludeNamespaces" -}}
+{{- $excludeNamespaceRegex := .Values.sumologic.metrics.excludeNamespaceRegex | quote -}}
+{{- if eq .Values.sumologic.collectionMonitoring false -}}
```

Comment:
why do we have this condition on `collectionMonitoring`

Reply:
Yeah this is a good point. We'd need to do this for both logs and metrics.